README.md

```diff
@@ -28,24 +30,28 @@ supply it when required. Utilities can accept this token in two ways:
 
 For example:
 
-    GITHUB_API_TOKEN=mytoken python3 biomage stage
+    GITHUB_API_TOKEN=mytoken biomage stage
 
 or
 
-    python3 biomage stage -t mytoken
+    biomage stage -t mytoken
 
 Using the environment variable means you can put the token in your
 `.bashrc` or `.zshrc` file, thereby avoiding typing it again and again. You can
 then simply do:
 
-    python3 biomage stage
+    biomage stage
 
 ### Other Enviroment Variables
 
 * `BIOMAGE_NICK` is optional and used to override the `USER` environment variable
   as the first part of the name of the staging environments created by you:
   `${BIOMAGE_NICK:-${USER}}-...`.
 
+*  `BIOMAGE_DATA_PATH`: where to get the experiment data to populate inframock's S3 and DynamoDB. If
```

Comment:
Can you provide a suggestion where this would preferably be set? Currently all the data is placed in `inframock`, in the future, would it be better to have the data under `biomage-utils` and point `inframock` to get the data from that folder?

Reply:
done